docs/changes/0.23.0.rst

```diff
@@ -0,0 +1,44 @@
+Changelog for QCoDeS 0.23.0
+===========================
+
+The February 2021 release of QCoDeS.
+
+---------
+Improved:
+---------
+
+- Add ``pandas`` and ``xarray`` methods to ``DataSetCache``  matching the ``DataSet`` (#2763)
+- Fix to Avoid loading data twice in dataset's ``to_xarray_dataset`` method (#2762)
+- Fix for the ``to_pandas`` method on a live dataset by not loading from disk (#2761)
+- Add optional ``tqdm`` progress bar to ``doNd`` functions (#2734)
+- Improvment to ``Station`` by adding the ``load_all_instruments`` method to load everything specified
+  in configuration yaml (#2721)
+- In ``guid_helpers`` the regular expression used to read guids from a string represantation of a
+  list/tuple of guids was flawed. Improvment to use ``ast.parse`` makes it a lot easier and error proof. (#2700)
+- Fix for station snapshot of parameters with ``snapshot_exclude`` (#2692)
+
+
+-----------------
+Improved drivers:
+-----------------
+
+- Feature to determine the validators for the frequency and power parameter of the Agilent ``E8527D` microwave
+  source based on the installed options (#2690)
+
+
+---------------
+Under the hood:
+---------------
+
+- Simplify science requirements tests (#2768)
+- Fix to use correct visa address for lakeshore pyvisa-sim in ``Station`` tests (#2754)
+- Update pip package caching since this key should depend on setup.cfg not setup.py (#2739)
+- Remove dependency on gitpython (#2738)
+- Fetch full git history when building docs so tag can be included in docs build info (#2733)
+- Add two more SMUs to B1500 yaml sim file, useful for testing (#2722)
+- Build docs on windows using github actions (#2716)
+- Move upload to pypi to Github actions (#2713)
+- Remove duplicate get call in ``_ParamCaller`` (#2712)
```

Comment:
This is an important bug fix as it slowed down multi threaded experiments with a factor of 2